docs/build-insights/reference/vcperf-commands.md

```diff
@@ -18,23 +18,23 @@ This article lists and describes the commands available in *`vcperf.exe`*, and h
 ## Commands to start and stop traces
 
 > [!IMPORTANT]
-> The following commands all require administrative privileges.
+> Unless /noadmin is provided, the following commands require administrative privileges.
```

Comment:
Consider adding this note "Unless /noadmin is provided" in the vcperf documentation on GitHub as well.